OSX/xcode/ags/plugin_registration.cpp

```diff
@@ -0,0 +1,34 @@
+//
+//  plugin_registration.cpp
+//  Shivah
+//
+//  Created by Nick Sonneveld on 26/06/2016.
+//
+//
+
+#include "plugin_registration.hpp"
+
+#include "agsplugin.h"
+//#include "example.h"
+
+typedef void (*t_engine_pre_init_callback)(void);
+extern void engine_set_pre_init_callback(t_engine_pre_init_callback callback);
+
+static void pl_register_builtin_plugin() {
+
+//  pl_register_builtin_plugin((InbuiltPluginDetails){
+//    .filename = "example",
```

Comment:
Oh, my mistake, I guess I did not use this kind of initialization for a long time.